internal/workload/podspec_updates.go

```diff
@@ -698,6 +710,14 @@ func (s *updateState) addHealthCheck(p *cloudsqlapi.AuthProxyWorkload, c *corev1
 	s.addProxyContainerEnvVar(p, "CSQL_PROXY_HTTP_PORT", fmt.Sprintf("%d", port))
 	s.addProxyContainerEnvVar(p, "CSQL_PROXY_HTTP_ADDRESS", "0.0.0.0")
 	s.addProxyContainerEnvVar(p, "CSQL_PROXY_HEALTH_CHECK", "true")
+	if adminPortPtr != nil {
```

Comment:
I think this should be reversed: if a user requests the quitquitquit endpoint or the debug endpoint, then the admin port becomes required. Enabling quitquitquit as a side-effect is a bit surprising.

Reply:
I removed the automatic quitquitquit configuration. We'll add it back in the future.